tools/sof-dump-status.py

```diff
@@ -2,10 +2,31 @@
 
 import subprocess
 import os
+import sys
+
 from common import format_pipeline, export_pipeline
 
+
+# i.MX specific maps
+IMX_SOC_ID_TO_FW_NAME = {"i.MX8QM": "imx8"}
+IMX_SOC_ID_TO_DSP_NAME = {"i.MX8QM": "556e8000.dsp"}
+IMX_SOC_ID_TO_FW_PATH = {"i.MX8QM": "/lib/firmware/imx/sof"}
+
+# maps containing the content of all platform-specific maps
+SOC_ID_TO_FW_NAME = {**IMX_SOC_ID_TO_FW_NAME}
+SOC_ID_TO_DSP_NAME = {**IMX_SOC_ID_TO_DSP_NAME}
+SOC_ID_TO_FW_PATH = {**IMX_SOC_ID_TO_FW_PATH}
```

Comment:
Maybe a bit shorter:
```suggestion
FW_PATHS = {**IMX_FW_PATHS}
```

It's a matter of taste but in general I find it enough to have only the "to" field in maps names because it's then invoked like this:
```
FW_PATHS[self.dt_info["soc_id"]]
```
so `soc_id` is already there.

No big deal.